Functions.Templates/Templates/DurableFunctionsEntity-Python/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "defaultFunctionName": "DurableFunctionsEntityPy",
+  "description": "$DurableFunctionsEntity_description",
+  "name": "Durable Functions entity",
+  "language": "Python",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_durableFunctions"
+  ],
+  "categoryStyle": "other",
+  "enabledInTryMode": false,
+  "userPrompt": [],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask",
+      "version": "2.3.0"
+    }
```

Comment:
Honestly unsure about what this block achieves, but I got it from @bachuv 's PR. Can anyone clarify what it does?

Reply:
I think this is used in some places where it automatically does `func extensions install` for you when you add the binding to a function (like the portal?). Not sure if this would be used anywhere for Python but @soninaren will know.